core/src/avm1/flv.rs

```diff
@@ -24,6 +24,17 @@ fn avm1_object_from_flv_variables<'gc>(
     info_object.into()
 }
 
+fn avm1_date_from_flv_date<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    unix_time: f64,
+    _local_offset: i16,
```

Comment:
So, I'd have to write tests to check for certain how local offset winds up getting used, if at all. However, given that this takes UNIX timestamps, which are always UTC by default, having a local offset is silly, so my money's on unused.